reposettings.py

```diff
@@ -169,21 +169,34 @@ def set(repo: Repository.Repository, config):
                 newsettings['dismiss_stale_reviews'] = bool(rules['dismiss-stale-reviews'])
             if 'required-review-count' in rules:
                 newsettings['required_approving_review_count'] = int(rules['required-review-count'])
-            if 'dismiss-pull-request-reviews' in rules:
-                if 'users' in rules['dismiss-pull-request-reviews']:
-                    newsettings['dismissal_users'] = rules['dismiss-pull-request-reviews']['users']
-                if 'teams' in rules['dismiss-pull-request-reviews']:
-                    newsettings['dismissal_teams'] = rules['dismiss-pull-request-reviews']['teams']
-                if 'apps' in rules['dismiss-pull-request-reviews']:
-                    newsettings['dismissal_apps'] = rules['dismiss-pull-request-reviews']['apps']
-            if 'allow-bypass-pull-request-reviews' in rules:
-                if 'users' in rules['allow-bypass-pull-request-reviews']:
-                    newsettings['users_bypass_pull_request_allowances'] = rules['allow-bypass-pull-request-reviews']['users']
-                if 'teams' in rules['allow-bypass-pull-request-reviews']:
-                    newsettings['teams_bypass_pull_request_allowances'] = rules['allow-bypass-pull-request-reviews']['teams']
-                if 'apps' in rules['allow-bypass-pull-request-reviews']:
-                    newsettings['apps_bypass_pull_request_allowances'] = rules['allow-bypass-pull-request-reviews']['apps']
```

Comment:
This would be a breaking change, right? Would it be too much effort to handle both old and new syntax? Something like we did in https://github.com/txqueuelen/reposettings/pull/8 (alhtough that was admittedly simpler).